pkg/cloud/services/networking/router.go

```diff
@@ -32,15 +32,15 @@ import (
 
 func (s *Service) ReconcileRouter(openStackCluster *infrav1.OpenStackCluster, clusterName string) error {
 	if openStackCluster.Status.Network == nil || openStackCluster.Status.Network.ID == "" {
-		s.scope.Logger().V(3).Info("No need to reconcile router since no network exists.")
+		s.scope.Logger().V(3).Info("No need to reconcile router since no network exists")
 		return nil
 	}
 	if len(openStackCluster.Status.Network.Subnets) == 0 {
-		s.scope.Logger().V(4).Info("No need to reconcile router since no subnet exists.")
+		s.scope.Logger().V(4).Info("No need to reconcile router since no subnet exists")
 		return nil
 	}
 	if openStackCluster.Status.ExternalNetwork == nil || openStackCluster.Status.ExternalNetwork.ID == "" {
-		s.scope.Logger().V(3).Info("No need to create router, due to missing ExternalNetworkID.")
+		s.scope.Logger().V(3).Info("No need to create router, due to missing ExternalNetworkID")
```

Comment:
We probably should rework the grammar of this while we're here. How about:

```
No need to reconcile router since no external network exists
```

?